### PR TITLE
Adds the AOE-generator  to the ships , adds it as a req purchaseable

### PR DESCRIPTION
**Pull request by MLGTASTICa**
## About The Pull Request

Adds 1 A.D.V.P.A.C.M.A.N mobile-power generator to each ship map in rotation , adds it to req , 1 generator for 20 points.
## Why It's Good For The Game

Less reliance on easy to saboutage APC's.

## Changelog
:cl:
add:  A.D.V.P.A.C.M.A.N AOE Generator on all ship maps.
add:  A.D.V.P.A.C.M.A.N as a purchaseable requisitions item.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -28. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/modules/reqs/supplypacks.dm [hunk 1]
```diff
@@ -1158,6 +1158,10 @@ ENGINEERING
 	name = "Combat Grade Floodlight"
 	contains = list(/obj/machinery/floodlightcombat)
 	cost = 5
+/datum/supply_packs/engineering/advanced_generator
+	name = "Wireless power generator"
```
Comment (XSlayer300) [inline code/modules/reqs/supplypacks.dm:1162]:
```suggestion
	name = "wireless power generator"
```

Comment (MLGTASTICa) [reply code/modules/reqs/supplypacks.dm:1162]: All of them start uppercase.

**Comment by Swagile** [comment]
please

APC's break sometimes when you build them too, as in certain areas don't allow APC's for some retarded reason

this fixes that

File: code/modules/reqs/supplypacks.dm [hunk 2]
```diff
@@ -1158,6 +1158,10 @@ ENGINEERING
 	name = "Combat Grade Floodlight"
 	contains = list(/obj/machinery/floodlightcombat)
 	cost = 5
+/datum/supply_packs/engineering/advanced_generator
```
Comment (TiviPlus) [inline code/modules/reqs/supplypacks.dm:1161]:
```suggestion
	cost = 5

/datum/supply_packs/engineering/advanced_generator
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Review by TiviPlus — APPROVED** [review]

**Review by SplinterGP — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-38). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**